Makefile

```diff
@@ -3,6 +3,6 @@ test:
 	go test -v ./...
 
 e2e:
-	cd ./e2e && go test -v ./...
+	cd ./e2e && go test -timeout=50m -v ./...
```

Comment:
If we only run the E2E tests in CI ```go test -v ./... -run TestE2e``` we should be able to remove this timeout. The Fuzz tests will not be tested on each PR but nightly most likely.